BuildResidentialHPXML/measure.rb

```diff
@@ -3409,26 +3415,32 @@ def self.set_hvac_distribution(hpxml, runner, args)
     end
 
     # Ducts
-    ducts_supply_location = args[:ducts_supply_location]
-    if ducts_supply_location == Constants.Auto
-      ducts_supply_location = get_duct_location_auto(args, hpxml)
+    if args[:ducts_supply_location] != Constants.Auto
+      ducts_supply_location = args[:ducts_supply_location]
+    end
+
+    if args[:ducts_return_location] != Constants.Auto
+      ducts_return_location = args[:ducts_return_location]
+    end
+
+    if args[:ducts_supply_surface_area] != Constants.Auto
+      ducts_supply_surface_area = args[:ducts_supply_surface_area]
     end
 
-    ducts_return_location = args[:ducts_return_location]
-    if ducts_return_location == Constants.Auto
-      ducts_return_location = get_duct_location_auto(args, hpxml)
+    if args[:ducts_return_surface_area] != Constants.Auto
+      ducts_return_surface_area = args[:ducts_return_surface_area]
```

Comment:
How does this work? Isn't it a string and needs to be converted to a float?

Reply:
And now the get_duct_location_auto method can be removed, right?

Reply:
I thought the hpxml.rb resource handled the string-to-float conversion?

Reply:
Yes, I will remove the get_duction_location_auto method.

Reply:
Oh, right. Good point. 🤦‍♂️